src/services/ai/mistral.service.ts

```diff
@@ -44,18 +44,24 @@ export interface CreateChatCompletionsResponse {
 }
 
 export class MistralService extends AIService {
-    private host = `https://api.mistral.ai`;
```

Comment:
Could you please use Camelcase for variables?

Reply:
Updated some snake_case names to camelCase.